doc/0000-ccl_note/main.tex

```diff
@@ -667,6 +674,84 @@ \subsubsection{Normalization of the power spectrum}
 used, only the option to specify $\sigma_8$ is supported. The computation of
 $\sigma_8$ is described in Section \ref{sec:hmf}.
 
+\subsection{Nonlinear Power Spectra}
+\label{sec:Nonlinear_Pk}
+
+Nonlinear perturbation theory (e.g.\ \cite{bernardeau02}) provides a method to calculate power spectra in the mildly nonlinear regime. {\tt CCL} provide sperturbation theory predictions for dark matter clustering, galaxy biasing, and intrinsic alignments using the {\tt FAST-PT} package \cite{mcewen16,fang17}. These power spectra can then be used for the relevant $P_{\alpha\beta}$, as described below in \ref{sec:cl}. Currently, all modeling is done at ``one-loop'' order, i.e.\ $\mathcal{O}(P_{\rm lin}^2)$, in standard perturbation theory (SPT). \ccl implements these models using {\tt PTTracer} objects.
+
+\subsubsection{Matter density}
+
+The matter density field can be expanded:
+\begin{equation}
+  \label{eq:delta_spt}
+  delta_m = \delta^{(1)} + \delta^{(2)} + \delta^{(3)} + \cdots,
```

Comment:
Typo here delta->\delta

Reply:
where? in the eq label, or somewhere else?

Reply:
Line 687, delta_m

Reply:
ah, yes. sorry - totally missed that. done now.